presentation/src/main/java/org/sagebionetworks/research/presentation/ActionType.java

```diff
@@ -37,9 +37,18 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * This interface defines the various types of actions that are possible to have.
+ */
 @Retention(RetentionPolicy.SOURCE)
-@StringDef({ActionType.FORWARD})
+@StringDef({ActionType.BACKWARD, ActionType.CANCEL, ActionType.FORWARD, ActionType.INFO, ActionType.LEFT_TAP,
+        ActionType.RIGHT_TAP, ActionType.SKIP})
 public @interface ActionType {
-    String FORWARD = "next";
     String BACKWARD = "back";
+    String CANCEL = "cancel";
+    String FORWARD = "next";
+    String INFO = "info";
+    String LEFT_TAP = "leftTap";
```

Comment:
LEFT_TAP and RIGHT_TAP should only be in mPower

Reply:
I'm going to migrate everything over to mpower after this pr so I think this would be better fixed with those changes.